components/modal/Register/register.component.js

```diff
@@ -0,0 +1,214 @@
+                            invalid={errors.reason ? true : false}
+                            name="reason" onChange={this.handleChange} />
+                        <span className={classNames.errorPrompt}><FormFeedback>{errors.reason}</FormFeedback></span>
+                    </FormGroup>
+                </div>
+                <br />
+
+                <FormGroup>
+                    <Label className={classNames.tagWord} for="email">Email Address</Label>
+                    <br />
+                    <Input id="email" value={recruiterData.email}
+                        className={classNames.inputBox}
+                        type="email"
+                        placeholder="someone@something.com"
+                        invalid={errors.email ? true : false}
+                        name="email" onChange={this.handleChange} />
+                    <span className={classNames.errorPrompt}><FormFeedback>{errors.email}</FormFeedback></span>
+                </FormGroup>
+                <br />
+
+                <FormGroup>
+                    <Label className={classNames.tagWord}>Currency</Label>
+                    <br />
+                    <div className="select">
+                        <Input type="select" name="currency" value={recruiterData.currency}
+                            className={classNames.inputBoxCur} invalid={errors.currency ? true : false} onChange={this.handleChange}>
+                            <option value="">--Select--</option>
+                            <option value="INR">Indian Rupee(INR)</option>
+                            <option value="USD">United States Dollar(USD)</option>
+                            <option value="EUR">Euro(EUR)</option>
+                            <option value="GBP">Great Britain Pound(GBP)</option>
+                            <option value="AUD">Australian Dollar(AUD)</option>
+                            <option value="CAD">Canadian Dollar(CAD)</option>
+                            <option value="SGD">Singapore Dollar(SGD)</option>
+                            <option value="JPY">Japanese Yen(JPY)</option>
+                        </Input>
+                        <span className={classNames.errorPrompt}><FormFeedback>{errors.currency}</FormFeedback></span>
+                    </div>
+                </FormGroup>
+                <br />
+
+                <FormGroup>
+                    <Label className={classNames.tagWord} for="package">Annual Package Offered</Label>
+                    <br />
+                    <Input className={classNames.inputBox} id="package" value={recruiterData.package}
+                        type="number"
+                        placeholder="2000000"
+                        invalid={errors.package ? true : false}
+                        name="package" onChange={this.handleChange} />
+                    <span className={classNames.errorPrompt}><FormFeedback>{errors.package}</FormFeedback></span>
+                </FormGroup>
```

Comment:
So much of repetition, can we refactor this? by moving them into components, like a `form` component